packages/web-client/src/components/comment_form.tsx

```diff
@@ -0,0 +1,100 @@
+import type { InterviewCommentModel, NewInterviewCommentModel } from '@job-seekr/data/validation'
+import React, { useState } from 'react'
+import { dateToTimestamp, timestampToISO } from '../utils'
+
+
+
+type AddInterviewProps = {
+  mode: 'add'
+  interview_id: string,
+  isBusy: boolean
+  onSubmit: (interview: NewInterviewCommentModel) => void
+  onCancel: () => void
+}
+
+
+type EditCommentProps = {
+  mode: 'edit',
+  isBusy: boolean
+  comment: InterviewCommentModel
+  onSubmit: (id: string, interview: NewInterviewCommentModel) => void
+  onCancel: () => void
+}
+
+type CommentFormProps = AddInterviewProps | EditCommentProps;
+
+type FormState =
+  Omit<NewInterviewCommentModel, 'comment_date'> & { comment_date: string };
+
+const getInitialFormState = (props: CommentFormProps): FormState => {
+  if (props.mode === 'add') {
+    var now = new Date();
+    now.setMinutes(now.getMinutes() - now.getTimezoneOffset());
+
+    return {
+      comment: '',
+      comment_date: now.toISOString().slice(0, 16),
```

Comment:
There is a function in `utils` that does a similar thing. I like the fact that you took the time zone into account, good thinking! Maybe, you could extend that function to incorporate this functionality?